lib/core/include/scipp/core/spatial_transforms.h

```diff
@@ -55,7 +60,7 @@ class Translation {
 public:
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
   Translation() : m_vec(Eigen::Vector3d(0, 0, 0)){};
-  explicit Translation(const Eigen::Vector3d &x) : m_vec(x){};
+  explicit Translation(Eigen::Vector3d x) : m_vec(std::move(x)){};
```

Comment:
Read https://eigen.tuxfamily.org/dox/TopicPitfalls.html and https://eigen.tuxfamily.org/dox/group__TopicPassingByValue.html. While `Vector3d` should be an exception, I would nevertheless be wary. I suspect the `std::move` may actually be a pessimisation?

Reply:
Wasn't aware of that. I just followed clang-tidy's suggestion.